weave/trace/op_type.py

```diff
@@ -355,7 +364,13 @@ def _get_code_deps(
         var_value = None
         if isinstance(fn, py_types.FunctionType):
             var_value = resolve_var(fn, var_name)
-        if var_value == fn:
+
+        try:
+            # Some objects throw on equality comparison (like dataframes)
+            equivalent = var_value == fn
+        except ValueError:
```

Comment:
Does pandas specifically throw `ValueError`?

Reply:
yeah, should we make this broader?

Reply:
Nah, it's probably ok -- we have the outer try/except that will catch other errors if any